samples/go/go.mod

```diff
@@ -0,0 +1,8 @@
+module leadIntegration
+
+go 1.16
+
+require (
+	github.com/gorilla/mux v1.8.0
```

Comment:
The classic :heart:

Reply:
Why not [Echo](https://github.com/labstack/echo)? which have some really great middlewares (ie. no need to implement BasicAuthentication)

Reply:
Because I already used gorilla/mux before, so i chose it for implement this sample

Reply:
mux is compatible with APM zap + tools. Echo has this support? or would have to implement?
Anyway, you could rewrite on another branch using Echo and show the pros and cons.

Reply:
Show me the code, dude! :)

Reply:
> Show me thecode